### PR TITLE
providing standard platform check utility

### PR DESCRIPTION
**Pull request by Bharath-2003**
To make use of standard utility function for platform check.

### CODE REVIEW

**Comment by merks** [comment]
@Bharath-2003 

You will need to sign the ECA for us to be able to accept contributions:

https://github.com/eclipse-platform/.github/blob/main/CONTRIBUTING.md#setting-up-your-eclipse-and-github-account

**Comment by Bharath-2003** [comment]
@merks I have completed the signing of ECA.

**Comment by merks** [comment]
All the builds fail like this:
```
06:32:23.862 [ERROR] Failed to execute goal org.eclipse.tycho:tycho-compiler-plugin:4.0.7:compile (default-compile) on project org.eclipse.search: Compilation failure: Compilation failure: 
06:32:23.862 [ERROR] /home/jenkins/agent/workspace/eclipse.platform.ui_PR-1818/bundles/org.eclipse.search/search/org/eclipse/search/internal/ui/text/FileLabelProvider.java:[22] 
06:32:23.862 [ERROR] 	import org.eclipse.swt.SWT;
06:32:23.863 [ERROR] 	       ^^^^^^^^^^^^^^^^^^^
06:32:23.863 [ERROR] The import org.eclipse.swt.SWT is never used
06:32:23.863 [ERROR] -> [Help 1]
```

File: bundles/org.eclipse.jface/src/org/eclipse/jface/viewers/FocusCellOwnerDrawHighlighter.java [hunk 1]
```diff
@@ -106,7 +106,7 @@ private void markFocusedCell(Event event, ViewerCell cell) {
 
 	private void removeSelectionInformation(Event event, ViewerCell cell) {
 		GC gc = event.gc;
-		if ("gtk".equals(SWT.getPlatform())) { //$NON-NLS-1$
+		if(Util.isGtk()){
```
Comment (merks) [inline bundles/org.eclipse.jface/src/org/eclipse/jface/viewers/FocusCellOwnerDrawHighlighter.java]: Please leave a space after the `if` and before the `{`.

File: bundles/org.eclipse.jface/src/org/eclipse/jface/viewers/FocusCellOwnerDrawHighlighter.java [hunk 2]
```diff
@@ -23,7 +23,7 @@
 import org.eclipse.swt.graphics.Rectangle;
 import org.eclipse.swt.widgets.Event;
 import org.eclipse.swt.widgets.Listener;
-
+import org.eclipse.jface.util.Util;
```
Comment (merks) [inline bundles/org.eclipse.jface/src/org/eclipse/jface/viewers/FocusCellOwnerDrawHighlighter.java]: Has Organize Imports been used?  It's odd that there is no blank line after the import.

File: bundles/org.eclipse.search/search/org/eclipse/search/internal/ui/text/FileLabelProvider.java [hunk 3]
```diff
@@ -186,7 +187,7 @@ private int appendShortenedGap(String content, int start, int end, int charsToCu
 
 
 	private int getCharsToCut(int contentLength, Match[] matches) {
-		if (contentLength <= 256 || !"win32".equals(SWT.getPlatform()) || matches.length == 0) { //$NON-NLS-1$
+		if (contentLength <= 256 || !(Util.isWin32()) || matches.length == 0) { //$NON-NLS-1$
```
Comment (merks) [inline bundles/org.eclipse.search/search/org/eclipse/search/internal/ui/text/FileLabelProvider.java]: Please change `!(Util.isWin32())` to be `!Util.isWin32()`

**Review by merks — CHANGES_REQUESTED** [review]

**Comment by merks** [comment]
I think you added an import that was already present.  Organize Imports should help avoid/fix that.

**Comment by merks** [comment]
I think you added an import that was already present.  Organize Imports should help avoid/fix that.

File: bundles/org.eclipse.jface/src/org/eclipse/jface/viewers/FocusCellOwnerDrawHighlighter.java (hunk 1 above)
Comment (Bharath-2003) [reply bundles/org.eclipse.jface/src/org/eclipse/jface/viewers/FocusCellOwnerDrawHighlighter.java]: done the changes.

File: bundles/org.eclipse.jface/src/org/eclipse/jface/viewers/FocusCellOwnerDrawHighlighter.java (hunk 2 above)
Comment (Bharath-2003) [reply bundles/org.eclipse.jface/src/org/eclipse/jface/viewers/FocusCellOwnerDrawHighlighter.java]: done the changes.

File: bundles/org.eclipse.search/search/org/eclipse/search/internal/ui/text/FileLabelProvider.java (hunk 3 above)
Comment (Bharath-2003) [reply bundles/org.eclipse.search/search/org/eclipse/search/internal/ui/text/FileLabelProvider.java]: done the changes.

**Review by merks — APPROVED** [review]
Looks fine.

**Comment by merks** [comment]
Thank you for contributing. 👍

**Pull request merged**